akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcSplitBrainSpec.scala

```diff
@@ -32,6 +28,7 @@ object MultiDcSplitBrainMultiJvmSpec extends MultiNodeConfig {
       akka.remote.netty.tcp.connection-timeout = 5 s # speedup in case of connection issue
       akka.remote.retry-gate-closed-for = 1 s
       akka.cluster.multi-data-center {
+        cross-data-center-gossip-probability = 0.8
```

Comment:
increasing too much will cause low probability for gossip within same dc, did you try with 0.5?

Reply:
I didn't, trying now

Reply:
0.5 is roughly the same as 0.8, whole test takes ~45s